### ISSUE TITLE
Cards Simple Vertical component heading needs ID to match aria-labelledby.

### ISSUE THREAD

**Issue by bs005600** [wvuweb/wvu-ds-v3-bookshop-components#58]
The heading ID to match the section's aria-lablledby is missing for the Cards Simple Vertical component.

**Example:** [https://important-hero.cloudvent.net/the-strategic-compass/](https://important-hero.cloudvent.net/the-strategic-compass/)

<img width="1679" height="445" alt="Image" src="https://github.com/user-attachments/assets/6430fd6e-b67d-419c-9708-aeb4120a8ac1" />
